lib/layouts.js

```diff
@@ -66,8 +66,8 @@ function basicLayout(loggingEvent) {
  * colouredLayout - taken from masylum's fork.
  * same as basicLayout, but with colours.
  */
-function colouredLayout(loggingEvent) {
-  return timestampLevelAndCategory(loggingEvent, loggingEvent.level.colour) + util.format(...loggingEvent.data);
+function colouredLayout(loggingEvent) {  
+  return timestampLevelAndCategory(loggingEvent, loggingEvent.level.colour) + util.formatWithOptions({depth: 4}, ...loggingEvent.data);
```

Comment:
@thegobot Your changes don't make it able to be changed, or am I missing something?